backend/handlers/handlers.go

```diff
@@ -686,6 +687,55 @@ func (handlers *Handlers) postDisconnectBaseHandler(r *http.Request) (interface{
 	return map[string]interface{}{"success": success}, nil
 }
 
+func (handlers *Handlers) postConnectElectrumHandler(r *http.Request) (interface{}, error) {
+	jsonBody := map[string]string{}
```

Comment:
When you move this to `bitboxbase/handlers`, don't forget to put the meat of the logic into `bitboxbase.go` and have the handler only do the http request parsing and response construction :smile: